samples/java/hibernate/src/main/resources/hibernate.properties

```diff
@@ -0,0 +1,11 @@
+hibernate.dialect org.hibernate.dialect.PostgreSQLDialect
+hibernate.connection.driver_class org.postgresql.Driver
+hibernate.connection.url jdbc:postgresql://localhost:5433/test-database
+hibernate.connection.username pratick
+
+hibernate.connection.pool_size 5
+
+hibernate.show_sql true
+hibernate.format_sql true
+
+hibernate.hbm2ddl.auto validate
```

Comment:
Does this really work with PGAdapter? That would actually be quite surprising (amazing)!

The current sample does not appear to include a script to create the data model. Would it be possible to modify the sample so it is auto-runnable, and the data model is created automatically the first time?

Reply:
Validate works, have tested it with a the test schema. 
Creating the schema using hibernate does not work.

Will add the SQL file in resources.